### DISCUSSION TITLE
get count with lazy load

### DISCUSSION THREAD

**Discussion by mkvcc** [adonisjs/core#2886] [Help] [answered]
Hi,
I need to get count for model relation with model object. so I cant use withCount, I need equivalent of loadCount for laravel. 
is there any way to do this is adonis?
appreciate any help. thanks

**Comment by arthurcvm** [comment]
Hi!
You can use `withCount` and the `computed` property, look this example:
In your queryBuilder you set this:
```ts
const users = await User.query().withCount('children') //Continue of your query...
```
In your model set this:
```ts
@computed()
  public get children_count (): number {
    return this.$extras.children_count
  }
```
and you has access of count in `users[index_of_user].children_count`

Docs references:
https://docs.adonisjs.com/reference/orm/query-builder#withcount
https://docs.adonisjs.com/reference/orm/decorators#computed

**Reply by mkvcc** [reply]
thanks. but the problem as i mentioned is that I cant use withCount. 
I need to get it with model object.

**Comment by thetutlage** [comment, marked as answer]
This is something not supported as of now. However, I will be working on it for the next release

**Reply by thetutlage** [reply]
Added in https://github.com/adonisjs/lucid/commit/a19b2326ffcddc42ea40824ee931e365da88c2cb
